src/components/App.js

```diff
@@ -60,6 +60,9 @@ const App = () => {
     const dispatch = useDispatch();
     const navigate = useNavigate();
 
+    useMetadataDependentData();
```

Comment:
is the metadata dependent data actually used here? it seems to be not related to the contents of this component